easybuild/easyconfigs/m/Mono/Mono-4.6.2.7.eb

```diff
@@ -0,0 +1,17 @@
+easyblock = 'ConfigureMake'
```

Comment:
My point is that we should avoid fragmentation where the existing software-specific easyblock is used in some easyconfigs, but not in others.

This is sort of a corner case though, since we can get away cleanly with using `ConfigureMake` if the Mono version is recent enough...

The changes that could/should be made to the Mono easyblock are basically to fall back to `ConfigureMake` for sufficiently recent versions of Mono.